docs/kqueen.rst

```diff
@@ -67,12 +69,12 @@ Example configuration files are located in `config/` directory. Default configur
       - Configuration file to load during startup
 
     * - LOG_LEVEL
-      - WARNING
-      - Logging level for `app.logger` (string)
+      - INFO
+      - Logging level for `kqueen_api` (string)
 
     * - SECRET_KEY
       - None
-      - This key is used for server-side encryption (cookies, secret database fields).
+      - This key is used for server-side encryption (cookies, secret database fields). Length must be at least 16 characters
```

Comment:
This key is used for server-side encryption (cookies, secret database fields) and must be at least 16 characters in length.